src/components/shared/nextButton.tsx

```diff
@@ -15,7 +20,11 @@ function NextButton(): JSX.Element {
   if (end) {
     return (<div></div>);
   } else {
-    return (<Link to={nextPage}  id='nextButton'>next</Link>);
+    return (
+      props.isEnabled ?
+        <Link to={nextPage} className='nextButton' id='enabled'onClick={props.disable} >next</Link>
+        :<Link to='' className='nextButton' onClick={(e) => e.preventDefault()} >next</Link>
+    );
```

Comment:
for accessibility reasons (and because it'd clean up this code), it'd be nice if you could make this a `button`!